WordPress/Classes/ViewRelated/Notifications/Views/LikeUserTableViewCell.xib

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Suggestion] This file is not needed now that the UI is created programmatically using SwiftUI. I suggest to remove this file and change cell registration to:

```swift
tableView.register(LikeUserTableViewCell.self, forCellReuseIdentifier: LikeUserTableViewCell.defaultReuseID)
```

Also, the `NibReusable` conformance should change to `Reusable`.

```
class LikeUserTableViewCell: UITableViewCell, Reusable
```